packages/ui/src/ui/UIFactory/default-ui-factory.tsx

```diff
@@ -319,4 +320,7 @@ export const defaultUIFactory: UIFactory = {
     renderMarkdown() {
         return undefined;
     },
+    getAnalyticsService(config) {
+        return [new YandexMetrika(config.metrikaCounterId)];
```

Comment:
It is better to keep YandexMetrika working if `config.metrikaCounterId` is provided